test/cartesian/Area.animation.spec.tsx

```diff
@@ -294,7 +329,10 @@ describe('Area animation', () => {
     it('should hide labels during the animation', async () => {
       const { container, animationManager } = renderTestCase();
 
-      return expectLabelsRemainVisible(container, animationManager);
```

Comment:
This test was wrong - expects labels to hide, but tests that they are visible? Fixed.